keel-sql/src/main/resources/db/changelog/20200714-add-event-application-column.yml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+- changeSet:
+    id: add-event-application-column
+    author: fletch
+    changes:
+    - sql:
+        sql: |
+          alter table event add column application varchar(255) generated always as (json ->> '$.application') stored;
```

Comment:
See https://www.compose.com/articles/mysql-for-json-generated-columns-and-indexing/

I'm not sure we _really_ need to use `stored` since the column will be part of an index. Any opinions?

Reply:
That link is currently borked for some reason, wayback has the article:

https://web.archive.org/web/20190718135741/https://www.compose.com/articles/mysql-for-json-generated-columns-and-indexing/

Reply:
I've changed it to virtual because we never actually select the value from that column, thus there's no point storing the value outside of the index. Testing locally with a dump of real data I could not see any significant difference in time to run the migration. Either way it was sub-second.